src/pages/home/report/ReportActionItem.js

```diff
@@ -334,6 +346,11 @@ function ReportActionItem(props) {
         return (
             <>
                 {children}
+                {!_.isEmpty(props.action.linkMetadata) && (
+                    <View style={props.draftMessage ? styles.chatItemReactionsDraftRight : {}}>
+                        <LinkPreviewer linkMetadata={_.filter(props.action.linkMetadata, (item) => !_.isEmpty(item))} />
```

Comment:
There are some empty items coming back? Maybe I can fix this as well on the backend.

Reply:
That was more to avoid error mentioned here: https://github.com/Expensify/App/pull/17288#issuecomment-1568177325

There should be array of objects in LinkPreviewer types, but if there is empty `[]` instead of real object it shows a warning of wrong types. Since _.isEmpty also removes empty arrays I used it to filter. 

One can say that:
`
            if (_.isArray(linkData)) {
                return;
            }
`
 inside LinkPreviewer is redundant but I just left it just to be safe.